dace/transformation/interstate/gpu_transform_sdfg.py

```diff
@@ -154,19 +160,42 @@ def can_be_applied(self, graph, expr_index, sdfg, permissive=False):
                     return False
         return True
 
-    def apply(self, _, sdfg: sd.SDFG):
+    def _get_marked_inputs_and_outputs(self, state, entry_node) -> list:
+        if (self.host_data is None or self.host_data == []) and (self.host_maps is None or self.host_maps == []):
+            return []
```

Comment:
shorter and more readable:
```suggestion
        if not self.host_data and not self.host_maps:
            return []
```